Letterbook.Adapter.TimescaleFeeds/FeedsAdapter.cs

```diff
@@ -131,22 +41,26 @@ public Task<int> RemoveFromTimelines(Models.Post post)
 		throw new NotImplementedException();
 	}
 
-	public IQueryable<Models.TimelineEntry> GetTimelineEntries(ICollection<Models.Audience> audiences, DateTime before,
-		int limit, bool includeBoosts = true)
+	public IQueryable<Models.Post> GetTimelineEntries(IEnumerable<Models.Audience> audiences, DateTimeOffset before,
+		int limit, bool includeShared = true)
 	{
-		var keys = audiences.Select(a => a.FediId.ToString());
-		return _feedsContext.Feeds
-			.AsNoTracking()
-			.Where(t => t.Time <= before)
-			.Where(t => keys.Contains(t.AudienceKey))
-			.Where(t => includeBoosts || t.BoostedBy != null)
-			.GroupBy(t => t.EntityId).Select(g => g.First())
-			// Equivalent to DistinctBy, but EFCore won't translate that for some reason
-			// .DistinctBy(t => t.EntityId)
-			.Take(limit);
+		var keys = audiences.Select(a => a.FediId);
+		return _feedsContext.Timelines
+				.AsNoTracking()
+				.OrderBy(t => t.Time)
```

Comment:
is EF Core smart enough to re-order this query? In older versions (framework era), this would prevent the query from translating correctly. The following "where" clauses would run in application code instead of SQL.